lib/src/kvs/ds.rs

```diff
@@ -485,15 +523,15 @@ impl Datastore {
 	) -> Result<(), Error> {
 		let dead_heartbeats = self.delete_dead_heartbeats(tx, watermark).await?;
 		trace!("Found dead hbs: {:?}", dead_heartbeats);
-		let mut archived: Vec<LqValue> = vec![];
+		let mut _archived: Vec<LqValue> = vec![];
 		for hb in dead_heartbeats {
 			let new_archived = self
 				.archive_lv_for_node(tx, &crate::sql::uuid::Uuid::from(hb.nd), this_node_id.clone())
 				.await?;
 			tx.del_nd(hb.nd).await?;
 			trace!("Deleted node {}", hb.nd);
-			for lq_value in new_archived {
-				archived.push(lq_value);
+			for _lq_value in new_archived {
+				// archived.push(lq_value);
```

Comment:
Can we just remove all these `_archived` and the for block and the commented-out `archived.push` if they are all unused?